debian/changelog

```diff
@@ -2,6 +2,8 @@ cloud-init (24.2-0ubuntu1~22.04.1) jammy; urgency=medium
 
   * d/control: remove netifaces due to GH-4634
   * d/p/deprecation-version-boundary.patch: Pin deprecation version to 22.1
+  * d/p/no-single-process.patch: Remove single process optimization
+  * d/p/no-nocloud-network.patch: Remove nocloud network feature
```

Comment:
Oops we also added this to the already released d/changelog entry, not the UNRELEASED entry. Will followup with a fix PR including both of these comments.